src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Microsoft.AspNetCore.Razor.LanguageServer.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)-windows</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
```

Reply:
Without this change, it doesn't build...
<img width="930" alt="image" src="https://user-images.githubusercontent.com/754264/152893812-65ce9dcc-8da6-4cc0-9738-5f5bfe731df8.png">